certification.md

```diff
@@ -40,19 +41,19 @@ These tests will be done during a 60-minute call with the Mews Marketplace team
 | **Add/remove a space and add/remove dates from the multi-room reservation** | Required | A modification push with the same [`channelManagerId`](mews-api.md#main-body) and Reservation [`code`](mews-api.md#reservation)s as the original reservation, and changes to 2 of the 3 bookings in the [`reservations`](mews-api.md#reservation) collection. | The channel manager receives a ```{ "clientToken":"[Mews Client Token]", "connectionToken":"[Token of a concrete connection]", "channelManagerId":"[Same id as the original push]", "reservations":[ {"code":"[Same code as the original push]", "confirmationNumber":"[Same confirmation number as the original push]"}]}``` push to the [Confirm Booking](channel-manager-api.md#confirm-booking) endpoint. |
 | **Cancel the multi-room reservation** | Required | An cancellation push with the same [`channelManagerId`](mews-api.md#main-body) as the original reservation and no additional data is sent to the [Process Group](mews-api.md#process-group) endpoint. | The channel manager receives a ```{ "clientToken":"[Mews Client Token]", "connectionToken":"[Token of a concrete connection]", "channelManagerId":"[Same id as the original push]", "reservations":[ {"code":"[Same code as the original push]", "confirmationNumber":"[Same confirmation number as the original push]"}]}``` push to the [Confirm Booking](channel-manager-api.md#confirm-booking) endpoint. |    |
-| **Receive 1 year of inventory data from Mews** | Required | The connection is enabled and Mews sends the next 365 days of inventory data in 26 consecutive pushes to the [Update Availability](channel-manager-api.md#update-availability), [Update Prices](channel-manager-api.md#update-prices) and [Update Restrictions](channel-manager-api.md#update-restrictions) endpoints.  | The channel manager returns ```{ "success": true}``` and data is updated in the channel manager extranet. | Being able to accept this data is required but push size or frequency can be decreased if the channel manager system can't accept it. |
+| **Send a reservation with a product attached** | Optional | A reservation push with a least 1 mapped product in the [`extra` collection](mews-api.md#extra) is sent to the [Process Group](mews-api.md#process-group) endpoint. | The channel manager receives a ```javascript { "clientToken":"[Mews Client Token]", "connectionToken":"[Token of a concrete connection]", "channelManagerId":"[Id sent in the reservation push]", "reservations":[ {"code":"[Code sent in the reservation push]", "confirmationNumber":"[Mews confirmation number]"}]}``` push to the [Confirm Booking](channel-manager-api.md#confirm-booking) endpoint. | These products must be manually mapped before certification. This test can be combined with the single reservation test. |
```

Comment:
I would also use better formatting of the code examples. e.g like we in connector doc.